proxy/certs/certs.go

```diff
@@ -182,6 +182,7 @@ func (s *RemoteCertSource) Local(instance string) (ret tls.Certificate, err erro
 		return err
 	})
 	if err != nil {
+		logging.Infof("createEphemeral failure \n")
```

Comment:
generally speaking, we try not to log anything, and this is sort of like the log and throw antipattern (here's an article about the java equivalent: https://community.oracle.com/docs/DOC-983543#logAndThrow)